packages/core/src/App/Containers/QuestionnaireModal/questionnaire-modal.tsx

```diff
@@ -29,13 +29,19 @@ const QuestionnaireModal = ({ ab_questionnaire, handleSignup }: TQuestionnaireMo
     }, [ab_questionnaire]);
 
     const onClickAnswer = (answer_code: string, answer_index: number) => {
-        Analytics.trackEvent('ce_questionnaire_form', {
-            action: 'choose_answer',
-            question: ab_questionnaire[0].question,
-            answer_content: ab_questionnaire[0].answers.filter(({ code }) => code === answer_code)[0].text,
-            answer_code,
-            answer_index,
-        });
+        window.dataLayer = [
+            ...window.dataLayer,
+            {
+                event: 'ce_questionnaire_form',
+                analyticsData: {
+                    action: 'choose_answer',
+                    question: ab_questionnaire[0].question,
+                    answer_content: ab_questionnaire[0].answers.filter(({ code }) => code === answer_code)[0].text,
+                    answer_code,
+                    answer_index,
+                },
+            },
+        ];
```

Comment:
why we need to set this data into window object ?

Reply:
to have an access to that global object in traders-hub (another page after registration)